redaxo/src/core/functions/function_rex_globals.php

```diff
@@ -19,7 +21,7 @@
  */
 function rex_get($varname, $vartype = '', $default = '')
```

Comment:
Was machen wir mit den globalen Funktionen? 
Behalten wir die und schieben die auch in den Namespace?

```php
<?php

use function Redaxo\Core\Http\get;

$foo = get('foo', 'int');
```

Sieht so bisschen komisch aus, oder? (einfach nur `get`) 
`Request::get('foo', 'int')` sieht mir da verständlicher aus.

Reply:
> Was machen wir mit den globalen Funktionen?

Würde ich weglassen und auf `Request::get('foo', 'int')` gehen.

Nur `Request::request('foo', 'int')` sieht dann wieder komisch aus.

Reply:
Wobei Symfony zum beispiel auch `$request->request->get('foo')` hat.